libs/watt/package/shell/nav-list/watt-nav-list.component.scss

```diff
@@ -21,19 +21,25 @@ watt-nav-list {
 
     a {
       @include watt.space-inset-squish-m;
+      padding: var(--watt-space-s);
       display: block;
       border-left: var(--watt-space-xs) solid transparent;
+      border-radius: var(--watt-space-s);
       color: var(--watt-color-neutral-white);
       text-decoration: none;
     }
 
+    a:hover {
+      transition: background-color linear 0.05s;
```

Comment:
suggestion: should we use the new animation variables?

Reply:
Fixed in https://github.com/Energinet-DataHub/greenforce-frontend/pull/4790/commits/f91b15a6ba7c6db7b965315a7cba717f9e412d22